runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -80,6 +80,9 @@
     long mNativeContent;
     long mNativeWebContents;
 
+    // TODO(boliu): This should be in a global context, not per webview.
+    private double mDIPScale = 1.0;
```

Comment:
setNativeContent will be always invokde. Why do we need to set the default value here?